tests/pose_estimation_pytorch/modelzoo/test_load_superanimal_models.py

```diff
@@ -0,0 +1,31 @@
+#
+# DeepLabCut Toolbox (deeplabcut.org)
+# © A. & M.W. Mathis Labs
+# https://github.com/DeepLabCut/DeepLabCut
+#
+# Please see AUTHORS for contributors.
+# https://github.com/DeepLabCut/DeepLabCut/blob/main/AUTHORS
+#
+# Licensed under GNU Lesser General Public License v3.0
+#
+import dlclibrary
+import pytest
+import torch
+
+from deeplabcut.pose_estimation_pytorch.modelzoo import get_super_animal_snapshot_path
+
+
+@pytest.mark.skip(reason="require-models")
```

Comment:
I ran these locally - but as it requires that all SuperAnimal models be downloaded locally, that would be too slow to run when the CI/CD pipeline is ran. You're correct that there's no CLI argument we can give to run this test. So the only way to run it again would be to remove the skip parameter.

The goal of this code was to test that the SuperAnimal models could all be loaded correctly with the `weights_only=True` option set. Now that it's be run once for all models it should be good (though I agree, it would be better to have a command-line option to be able to run it periodically). But `pytest` isn't great with that, so there isn't an easy way to do it.